jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/settings/MobileSettings.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.jdi.light.mobile.settings;
+
+import com.epam.jdi.light.settings.WebSettings;
+
+import static com.epam.jdi.light.driver.get.DriverInfos.*;
+import static com.epam.jdi.light.driver.get.RemoteDriver.*;
+import static com.epam.jdi.light.mobile.driver.MobileDriverData.*;
+import static com.epam.jdi.light.mobile.driver.MobileDriverInfos.*;
+import static com.epam.jdi.light.settings.WebSettings.*;
+import static com.epam.jdi.tools.PropertyReader.*;
+
+/**
+ * Created by Roman Iovlev on 20.03.2019
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+public class MobileSettings {
+    public static synchronized void init() {
+        if (initialized) return;
+        WebSettings.init();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'WebSettings.init' due to existing static import 'com.epam.jdi.light.settings.WebSettings.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)